client/webserver/site/src/js/locales.ts

```diff
@@ -210,7 +214,7 @@ export const zhCN: Locale = {
   [ID_API_ERROR]: '接口错误',
   [ID_ADD]: '加',
   [ID_CREATE]: '创建',
-  [ID_WALLET_READY]: 'Choose Wallet', // xxx translate
+  [ID_WALLET_READY]: 'Ready', // xxx translate
   [ID_SETUP_WALLET]: 'Setup' // xxx translate
```

Comment:
Can we delete these `xxx translate` entries? We already fall back to English, and having these entries inhibits our ability to generate a list of missing translations.